### ISSUE TITLE
Ios issue : not work on ios

### ISSUE THREAD

**Issue by mohu2212** [mohamedegy107/youtube_player_embed#2]
Not work on ios simulator even thumbline not appear

**Comment by mohamedegy107** [comment]
today i will fix all issues , try it at night and will fixed god willing

**Comment by mohamedegy107** [comment]
send screenshot

**Comment by mohu2212** [comment]
![Simulator Screenshot - iPhone 16 Pro Max - 2025-01-01 at 10 52 55](https://github.com/user-attachments/assets/09d90d08-df24-44ba-b277-033fb90d2a8f)


`[IOSInAppWebViewWidget] (iOS) IOSInAppWebViewWidget ID 0 calling "onWebViewCreated" using []
[IOSInAppWebViewController] (iOS) WebView ID 0 calling "onUpdateVisitedHistory" using {url: https://www.youtube.com/embed/bgzbWnwB6mg?autoplay=1&mute=0&controls=1&rel=0, isReload: null}
[IOSInAppWebViewController] (iOS) WebView ID 0 calling "shouldOverrideUrlLoading" using {sourceFrame: {securityOrigin: {protocol: , host: , port: 0}, isMainFrame: false, request: null}, isRedirect: null, navigationType: -1, isForMainFrame: true, targetFrame: {request: {cachePolicy: 0, url: , httpShouldHandleCookies: true, headers: {}, mainDocumentURL: null, networkServiceType: 0, allowsCellularAccess: true, timeoutInterval: 2147483647.0, httpShouldUsePipelining: true, allowsConstrainedNetworkAccess: true, method: GET, attribution: 0, body: null, allowsExpensiveNetworkAccess: true, assumesHTTP3Capable: false}, securityOrigin: {host: , port: 0, protocol: }, isMainFrame: true}, request: {method: GET, allowsCellularAccess: true, headers: {Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8, User-Agent: Mozilla/5.0 (iPhone; CPU iPhone OS 18_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148}, cachePolicy: 0, allowsExpensiveNetworkAccess: true, url: https://www.youtube.com/embed...
[IOSInAppWebViewController] (iOS) WebView ID 0 calling "onProgressChanged" using {progress: 0}
[IOSInAppWebViewController] (iOS) WebView ID 0 calling "onConsoleMessage" using {messageLevel: 3, message: TypeError: null is not an object (evaluating 'document.querySelector('.ytp-title').style')}
[IOSInAppWebViewController] (iOS) WebView ID 0 calling "onConsoleMessage" using {messageLevel: 3, message: TypeError: null is not an object (evaluating 'document.querySelector('.ytp-title-channel').style')}
[IOSInAppWebViewController] (iOS) WebView ID 0 calling "onContentSizeChanged" using {newContentSize: {width: 980.0, height: 551.0}, oldContentSize: {height: 0.0, width: 0.0}}
[IOSInAppWebViewController] (iOS) WebView ID 0 calling "onReceivedServerTrustAuthRequest" using {protectionSpace: {port: 443, receivesCredentialSecurely: true, protocol: https, proxyType: null, sslError: {code: 4, message: Indicates the evaluation succeeded and the certificate is implicitly trusted, but user intent was not explicitly specified.}, sslCertificate: {issuedBy: null, issuedTo: null, x509Certificate: [48, 130, 14, 10, 48, 130, 12, 242, 160, 3, 2, 1, 2, 2, 17, 0, 252, 18, 91, 189, 91, 54, 234, 107, 18, 59, 10, 85, 73, 136, 76, 159, 48, 13, 6, 9, 42, 134, 72, 134, 247, 13, 1, 1, 11, 5, 0, 48, 59, 49, 11, 48, 9, 6, 3, 85, 4, 6, 19, 2, 85, 83, 49, 30, 48, 28, 6, 3, 85, 4, 10, 19, 21, 71, 111, 111, 103, 108, 101, 32, 84, 114, 117, 115, 116, 32, 83, 101, 114, 118, 105, 99, 101, 115, 49, 12, 48, 10, 6, 3, 85, 4, 3, 19, 3, 87, 82, 50, 48, 30, 23, 13, 50, 52, 49, 50, 48, 50, 48, 56, 51, 53, 53, 55, 90, 23, 13, 50, 53, 48, 50, 50, 52, 48, 56, 51, 53, 53, 54, 90, 48, 23, 49, 21, 48, 19, 6, 3, 85, 4, 3, 12, 12, 42...
`

**Comment by mohamedegy107** [comment]
did you added the following to your Info.plist?

<key>NSAppTransportSecurity</key>
<dict>
  <key>NSAllowsArbitraryLoads</key>
  <true/>
</dict>

and added flutter_inappwebview to your dependencies ?

**Comment by mohu2212** [comment]
Yes

**Comment by kcs-govindtank** [comment]
<key>NSAppTransportSecurity</key>
	<dict>
		<key>NSAllowsArbitraryLoads</key>
		<true/>
		<key>NSAllowsArbitraryLoadsForMedia</key>
		<true/>
		<key>NSAllowsArbitraryLoadsInWebContent</key>
		<true/>
	</dict>
	
	
	@mohamedegy107  these above are added in my info.plist, still geting the blank white screen and no youtube video play

**Comment by mohu2212** [comment]
Didn't work

**Comment by mohamedegy107** [comment]
I fixed this , today i will upload the update and everything will working god willing

**Comment by kcs-govindtank** [comment]
@mohamedegy107  yes, in iOS, I have tested it with version 1.6.3 now the videos are playing
